api/tests/identity-access-management/unit/scripts/anonymize-users-in-batch.script.test.js

```diff
@@ -0,0 +1,88 @@
+import { AnonymizeUsersInBatchScript } from '../../../../src/identity-access-management/scripts/anonymize-users-in-batch.script.js';
+import { expect, sinon } from '../../../test-helper.js';
+
+describe('Unit | Identities Access Management | Scripts | Anonymize users in batch', function () {
+  let script;
+  let logger;
+  let anonymizeUser;
+
+  beforeEach(function () {
+    script = new AnonymizeUsersInBatchScript();
+    logger = {
+      info: sinon.spy(),
+      error: sinon.spy(),
+    };
+    anonymizeUser = sinon.stub();
+  });
+
+  describe('#handle', function () {
+    context('dryRun mode', function () {
+      it('should display the dryRun message', async function () {
```

Comment:
Ne pas utiliser should mais plutôt le comportement attendu du point testé : 
```suggestion
      it('displays the dryRun message', async function () {
```